docs/reference/migration/migrate_8_4.asciidoc

```diff
@@ -9,7 +9,7 @@ your application to {es} 8.4.
 
 See also <<release-highlights>> and <<es-release-notes>>.
 
-coming::[8.4.0]
+coming::[8.4.1]
```

Comment:
We'll need a follow-up PR to remove the `coming` tag when 8.4.1 is GA.

Reply:
We should just remove the coming tag, it doesn't need to be updated to 8.4.1.

Reply:
I removed the `coming` tag.